<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mafMainWindow</class>
 <widget class="QMainWindow" name="mafMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>821</width>
    <height>628</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>821</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>mafMainWindow</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>821</width>
      <height>451</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QMdiArea" name="mdiArea"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>821</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>35</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockSideBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>200</width>
     <height>42</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="dockWidgetArea">
    <number>2</number>
   </attribute>
   <widget class="QWidget" name="sideBarDockContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>201</width>
       <height>421</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="layoutSideBar">
      <item row="0" column="0">
       <widget class="QTabWidget" name="tabWidget">
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tabTree">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Tab that shows data hierarchy and property panel</string>
         </property>
         <attribute name="title">
          <string>Tree</string>
         </attribute>
         <widget class="QWidget" name="gridLayoutWidget_3">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>191</width>
            <height>391</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="layoutTree">
           <item row="0" column="0">
            <widget class="QWidget" name="hierarchyWidget" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <widget class="QWidget" name="gridLayoutWidget_7">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>0</y>
                <width>191</width>
                <height>201</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="layoutHierarchy"/>
             </widget>
             <widget class="QWidget" name="gridLayoutWidget_8">
              <property name="geometry">
               <rect>
                <x>0</x>
                <y>200</y>
                <width>195</width>
                <height>191</height>
               </rect>
              </property>
              <layout class="QGridLayout" name="layoutPropertiesBox">
               <item row="0" column="0">
                <widget class="QTabWidget" name="tabProperty">
                 <property name="tabShape">
                  <enum>QTabWidget::Rounded</enum>
                 </property>
                 <property name="currentIndex">
                  <number>0</number>
                 </property>
                 <property name="usesScrollButtons">
                  <bool>true</bool>
                 </property>
                 <property name="documentMode">
                  <bool>false</bool>
                 </property>
                 <property name="tabsClosable">
                  <bool>false</bool>
                 </property>
                 <property name="movable">
                  <bool>false</bool>
                 </property>
                 <widget class="QWidget" name="vmeTab">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <attribute name="title">
                   <string>vme</string>
                  </attribute>
                  <widget class="QWidget" name="gridLayoutWidget_9">
                   <property name="geometry">
                    <rect>
                     <x>9</x>
                     <y>9</y>
                     <width>181</width>
                     <height>161</height>
                    </rect>
                   </property>
                   <layout class="QGridLayout" name="vmeLayout"/>
                  </widget>
                 </widget>
                 <widget class="QWidget" name="visualPipeTab">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <attribute name="title">
                   <string>visual pipe</string>
                  </attribute>
                  <widget class="QWidget" name="gridLayoutWidget_10">
                   <property name="geometry">
                    <rect>
                     <x>-1</x>
                     <y>-1</y>
                     <width>191</width>
                     <height>161</height>
                    </rect>
                   </property>
                   <layout class="QGridLayout" name="visualPipeLayout"/>
                  </widget>
                 </widget>
                 <widget class="QWidget" name="dataPipeTab">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <attribute name="title">
                   <string>data pipe</string>
                  </attribute>
                  <widget class="QWidget" name="gridLayoutWidget_11">
                   <property name="geometry">
                    <rect>
                     <x>-1</x>
                     <y>-1</y>
                     <width>191</width>
                     <height>161</height>
                    </rect>
                   </property>
                   <layout class="QGridLayout" name="dataPipeLayout"/>
                  </widget>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
        <widget class="QWidget" name="tabView">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Tab that shows View's GUI</string>
         </property>
         <attribute name="title">
          <string>View</string>
         </attribute>
         <widget class="QWidget" name="gridLayoutWidget_4">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>191</width>
            <height>391</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="layoutView"/>
         </widget>
        </widget>
        <widget class="QWidget" name="tabOperation">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="toolTip">
          <string>Tab that shows Operation's GUI</string>
         </property>
         <attribute name="title">
          <string>Operation</string>
         </attribute>
         <widget class="QWidget" name="gridLayoutWidget_5">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>191</width>
            <height>391</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="layoutOperation"/>
         </widget>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QDockWidget" name="dockLogBarWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>79</width>
     <height>70</height>
    </size>
   </property>
   <property name="allowedAreas">
    <set>Qt::BottomDockWidgetArea|Qt::TopDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string/>
   </property>
   <attribute name="dockWidgetArea">
    <number>8</number>
   </attribute>
   <widget class="QWidget" name="logBarWidgetContents">
    <property name="sizePolicy">
     <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
      <horstretch>0</horstretch>
      <verstretch>0</verstretch>
     </sizepolicy>
    </property>
    <widget class="QWidget" name="gridLayoutWidget_6">
     <property name="geometry">
      <rect>
       <x>620</x>
       <y>0</y>
       <width>199</width>
       <height>41</height>
      </rect>
     </property>
     <layout class="QGridLayout" name="gridLayoutLogBar">
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <widget class="QWidget" name="findWidget" native="true">
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>1</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>1</number>
         </property>
         <item>
          <widget class="QToolButton" name="clearButton">
           <property name="text">
            <string>Clear</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="findEdit">
           <property name="text">
            <string>Find..</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
